docs/docusaurus/sidebars.json

```diff
@@ -274,6 +274,8 @@
         "label": "Access Gateway",
         "ids": [
           "lte/dev_notes",
+          "lte/openvswitch",
+          "lte/pipelined",
```

Comment:
Hm?

Reply:
It might be auto added file, reverted it. By mistake clicked on resolve conversation, sorry for that.